### ISSUE TITLE
Peeps stack in queue when having multiple entrances on one station

### ISSUE THREAD

**Issue by EmielRCT** [OpenRCT2/OpenRCT2#11549]
When having multiple entrances on one station, peeps will pile up in the queue till the max of 1000 guests in the queue is reached.

See it in action here: [https://youtu.be/rAQ7qI9VI1A](url)

Steps to reproduce:
-Build a ride
-Build an entrance and queue path to it
-Copy/paste the entrance to another place and make it useable via the tile inspector
-Build the second queue path

Having this fixed would be a nice addition to the game so you could make a more realistic station and queue, for example a fast pass section.

Taken in singleplayer, v.0.2.6-366-29fcdb5 (latest develop version)

**Comment by frutiemax** [comment]
I believe this is normal this leaves to abnormal behaviours due to the structure of the ride state.

```
struct RideStation
{
    CoordsXY Start;
    uint8_t Height;
    uint8_t Length;
    uint8_t Depart;
    uint8_t TrainAtStation;
    TileCoordsXYZD Entrance;
    TileCoordsXYZD Exit;
    int32_t SegmentLength; // Length of track between this station and the next.
    uint16_t SegmentTime;  // Time for train to reach the next station from this station.
    uint8_t QueueTime;
    uint16_t QueueLength;
    uint16_t LastPeepInQueue;

    static constexpr uint8_t NO_TRAIN = std::numeric_limits<uint8_t>::max();

    int32_t GetBaseZ() const;
    void SetBaseZ(int32_t newZ);
    CoordsXYZ GetStart() const;
};
```

I think the fact there is only one TileCoord for the entrance might affect the save file format.

**Comment by duncanspumpkin** [comment]
Yeah as soon as you use the tile inspector to move, copy things your going to get odd things happening
